### PR TITLE
Flyway 5.0.7 patches

### PR DESCRIPTION
**Pull request by MikeMostovyi**
Adapted project for NGTI Sonatype
Added patches for Flyway 5.0.7

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -132,18 +132,18 @@
...
+                        <!--<version>2.9.1</version>-->
+                        <!--<executions>-->
+                            <!--<execution>-->
+                                <!--<id>attach-javadocs</id>-->
+                                <!--<goals>-->
+                                    <!--<goal>jar</goal>-->
+                                <!--</goals>-->
+                            <!--</execution>-->
+                        <!--</executions>-->
+                    <!--</plugin>-->
```
Comment (gmbett) [inline pom.xml]: Remove commented code ?

File: pom.xml [hunk 2]
```diff
@@ -43,8 +43,8 @@
     </developers>
     <distributionManagement>
         <repository>
-            <id>bintray</id>
-            <url>https://api.bintray.com/maven/yanaga/maven/sqldroid</url>
+            <id>ngti.sonatype</id>
+            <url>https://repo.ngti.nl/nexus/content/repositories/snapshots/</url>
```
Comment (gmbett) [inline pom.xml]: Are you going to upload only snapshots ? I think we should make a release to prevent changes in the library.

File: pom.xml (hunk 1 above)
Comment (MikeMostovyi) [reply pom.xml]: Actually, I do not need to comment it. Compilation did not work at some point with this target, but now is fixed. Uncommented.

File: pom.xml (hunk 2 above)
Comment (MikeMostovyi) [reply pom.xml]: Changed to release.

**Review by gmbett — APPROVED** [review]

**Pull request merged**